docs/troubleshooting.md

```diff
@@ -0,0 +1,94 @@
+# Troubleshooting 
+
+Deployment failures at Kubernetes are cryptic and not intuitive for debugging. Users need to identify the errors from pod logs, pod events(describe pod), pod status, etc. However, this does not help to troubleshoot the error. When issues occur, abstraction is necessary for users to simplify troubleshooting.
+
+The following flow chart explains the troubleshooting flow for any hyscale deployment.
```

Comment:
Isn't this the troubleshooting flow for any app deployment to k8s? (with/without hyscale)